twine/commands/upload.py

```diff
@@ -18,7 +18,9 @@
 from twine.package import PackageFile
 from twine import exceptions
 from twine import settings
-from twine import utils
+from twine.utils import DEFAULT_REPOSITORY, TEST_REPOSITORY
```

Comment:
Same here. You were already importing `utils` as a module let's not regress that. I see above someone started importing PackageFile directly but I'll go through and fix that separately (or someone else can)